ci-operator/step-registry/gather/aws-console/gather-aws-console-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: gather-aws-console
+  from: upi-installer
+  commands: gather-aws-console-commands.sh
+  env:
+  - name: TMPDIR
+    default: /tmp
+    documentation: A pathname of a directory made available for programs that need a place to create temporary files.
```

Comment:
Same thought as https://github.com/openshift/release/pull/9640#discussion_r441911360

I think exposing TMPDIR as a user option is mis use of setting variables for the bash script.